_site/index.html

```diff
@@ -109,9 +109,9 @@ <h3>
           </h3>
           <p>
             TC39 welcomes contributions from the JavaScript community, whether it is feedback on existing proposals, improved documentation, testing, implementations, or even language feature ideas. <!-- TODO: Make each of the things in the preceding sentence links to the appropriate guides -->
-            See our <a href="https://github.com/tc39/ecma262/blob/master/CONTRIBUTING.md">contributor guide</a> for details. 
+            See our <a href="https://github.com/tc39/ecma262/blob/master/CONTRIBUTING.md">contributor guide</a> for details.
           </p><p>
-            To participate as a TC39 delegate in meetings, <a href="https://ecma-international.org/memento/join.htm">join Ecma</a> and sign our <a href="https://ecma-international.org/memento/Policies/Registration%20Form%20for%20the%20Ecma%20Royalty%20Free%20(RF)%20Task%20Group%20(TG).pdf">patent agreement</a>.
+            To participate in TC39 meetings, <a href="https://ecma-international.org/memento/join.htm">join Ecma</a> and sign our <a href="https://ecma-international.org/memento/Policies/Registration%20Form%20for%20the%20Ecma%20Royalty%20Free%20(RF)%20Task%20Group%20(TG).pdf">patent agreement</a>.
```

Comment:
this might seem a bit exclusionary, also its possible to participate in meetings as an observer without joining ecma... this seems to be very specific to becoming a member?

except for the patent agreement, probably that is necessary for anyone joining a meeting

Reply:
For better or worse, the current typical way of joining TC39 meetings is to join Ecma. If anyone is not sure whether they want to join Ecma and want more information about TC39 meetings, they can contact me at littledan@igalia.com; we can document this sort of thing when we have a "joining" guide.